property-set/docs/cloudshell-tutorial.md

```diff
@@ -0,0 +1,127 @@
+# Property Set
+
+---
+This sample shows how to easily get data from a Property Set using an AssignMessage policy.
```

Comment:
The same also gets data using JS policy. 
So lets copy the same description from the samples README here as well

Reply:
Done